schema.graphql

```diff
@@ -1,3 +1,139 @@
+# ADD TOTAL TOKENS
+# spread fees are expresed in sell token
+# write a how to query on a specific block
+
+
+######################################################################################################################
+type CowSwap @entity {
+  "Singleton #1"
+  id: ID!
+  "Total token ever traded"
+  tokensTotal: BigInt!
+  "Total orders placed"
+  ordersTotal: BigInt!
+  "Total unique addresses on cowsap"
+  usersTotal: BigInt!
+  "Total different settlements"
+  settlementTotal: BigInt!
+  "Total traded amount in USD"
+  totalTradedAmountUsd: BigDecimal
+  "Total traded amount in ETH"
+  totalTradedAmountEth: BigDecimal
+  "Total fee amount in USD"
+  totalFeeAmountUsd: BigDecimal
+  "Total fee amount in Eth"
+  totalFeeAmountEth: BigDecimal
+}
+
+######################################################################################################################
+type CowSwapDayData @entity {
+  "Day timestamp"
+  id: ID!
+  "Start day timestamp"
+  timestamp: BigInt!
```

Comment:
```suggestion
  dayTimestamp: BigInt!
```

Reply:
Do we need it if it is already the ID? happy to leave it for clarity if that's the reason

Reply:
I prefer to leave it for clarity. Id is a way to identify the entity and timestamp will be the information. 
About the suggestion I prefer to be polymorphic with the hourly version. But let me know if you still want to change it.